python/MooseDocs/extensions/MooseTextPatternBase.py

```diff
@@ -109,6 +114,7 @@ def createElement(self, label, content, filename, rel_filename, settings):
         code = etree.SubElement(pre, 'code')
         if settings['language']:
             code.set('class', settings['language'])
-        code.text = self.markdown.htmlStash.store(content.strip('\n'), safe=True)
+        content = content.replace('<', '&lt;').replace('>', '&gt;')
```

Comment:
Hmm, You shouldn't have to do this manually. You are only scratching the surface of the number of replacements that you may need to make in HTML content. There's should be an encoding method with the tools you are using, there always is. Otherwise you'll be in here in a few more weeks adding the next exception to this list.